### PR TITLE
annotation: refresh comments on accept/reject all tracked changes

### PR DESCRIPTION
**Pull request by lpranam**
problem:
tracked comments deletion was not reflected with accept/reject all changes

Change-Id: I0964fbd220f450cb144a115e8203250467e7edae

* Target version: master 

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Comment by lpranam** [comment]
requires: https://github.com/CollaboraOnline/online/pull/8127

File: browser/src/control/Control.NotebookbarWriter.js [hunk 1]
```diff
@@ -2283,8 +2283,8 @@ L.Control.NotebookbarWriter = L.Control.Notebookbar.extend({
 						'type': 'toolbox',
 						'children': [
 							{
-								'id': 'review-accept-all-tracked-changes',
-								'type': 'toolitem',
+								'id': 'acceptalltrackedchanges',
+								'type': 'menubartoolitem',
```
Comment (eszkadev) [inline browser/src/control/Control.NotebookbarWriter.js:2287]: should be `customtoolitem` so it will use `dispatch` not `menubar` code

Comment (eszkadev) [reply browser/src/control/Control.NotebookbarWriter.js:2287]: see https://github.com/CollaboraOnline/online/issues/8029

**Review by eszkadev — CHANGES_REQUESTED** [review]

File: browser/src/layer/tile/CommentListSection.ts [hunk 2]
```diff
@@ -1992,6 +1992,16 @@ export class CommentSection extends CanvasSectionObject {
 			comment.onCommentDataUpdate();
 		}
 	}
+
+	public rejectAllTrackedCommentChanges(): void {
+		for (var i = 0; i < this.sectionProperties.commentList.length; i++) {
+			var comment = this.sectionProperties.commentList[i];
+			if (comment.sectionProperties.data.layoutStatus === 3) {
+				comment.sectionProperties.data.layoutStatus = 2;
```
Comment (eszkadev) [inline browser/src/layer/tile/CommentListSection.ts:2000]: what are these magic numbers?
best to have them named by some static const in the class

Comment (lpranam) [reply browser/src/layer/tile/CommentListSection.ts:2000]: it is explained here in the comments: https://github.com/CollaboraOnline/online/blob/71a427c465c3b952f032b283cedc9e52ea985fcd/browser/src/layer/tile/CommentSection.ts#L69

but yes good to have some class

**Comment by eszkadev** [comment]
Would be good to clearly use enum also when assigning the value:
https://github.com/CollaboraOnline/online/blob/71a427c465c3b952f032b283cedc9e52ea985fcd/browser/src/layer/tile/CommentListSection.ts#L1219

**Review by eszkadev — APPROVED** [review]

**Pull request merged**